ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -2525,6 +2525,10 @@ msgstr "Visualisez les données"
 msgid "Access data"
 msgstr "Accédez aux données"
 
+#: ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html:68
+msgid "Access <abbr title=\"Artificial Intelligence\">AI</abbr>"
+msgstr "Accédez à l’<abbr title =\"intelligence artificielle\">IA</abbr>"
```

Comment:
Should the french title be capitalized?

Reply:
No and neither should the English because are using sentence case